app/routes/cookies.js

```diff
@@ -19,7 +20,7 @@ module.exports = [{
       payload: joi.object({
         analytics: joi.boolean(),
         async: joi.boolean().default(false)
-      })
+      }).unknown()
```

Comment:
Fix issue with not being able to update the cookie preferences due to the `crumb` being included in the payload.